watertap/property_models/NaCl_prop_pack.py

```diff
@@ -668,17 +670,17 @@ def rule_visc_d_phase(b):  # dynamic viscosity, eq 5 in Bartholomew
 
         self.eq_visc_d_phase = Constraint(rule=rule_visc_d_phase)
 
-    def _diffus_phase(self):
-        self.diffus_phase = Var(
+    def _diffus_phase_comp(self):
+        self.diffus_phase_comp = Var(
             self.params.phase_list,
```

Comment:
SHould also be indexed by component, even though this is the NaCl package

Reply:
Done